src/main/java/cn/nukkit/level/generator/Normal.java

```diff
@@ -37,6 +46,26 @@
  * HugeTreesGenerator.java
  */
 public class Normal extends Generator {
+    private static final int SMOOTH_SIZE = 3;
+    private static final Int2ObjectMap<Int2DoubleMap> GAUSSIAN_KERNEL;
+
+    static {
+        GAUSSIAN_KERNEL = new Int2ObjectOpenHashMap<>();
+        double bellSize = 1 / (double) SMOOTH_SIZE;
+        double bellHeight = 2 * (double) SMOOTH_SIZE;
+
+        for (int sx = -SMOOTH_SIZE; sx <= SMOOTH_SIZE; sx++) {
```

Comment:
This is code from pmmp waw

Reply:
oh, i added that in for testing and forgot to delete it. thanks!

(it's not actually used anywhere)

Reply:
Ok :d